src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,140 @@
+import * as Polymorphic from '@radix-ui/react-polymorphic'
+import React, { forwardRef } from 'react'
+import { CSS, StitchesVariants, styled } from 'stitches.config'
+import {
+  Root,
+  Trigger,
+  Content,
+  Item,
+  CheckboxItem,
+  ItemIndicator,
+} from '@radix-ui/react-dropdown-menu'
+import { Slot } from '@radix-ui/react-slot'
+import { Check } from 'index'
+
+const itemStyles = {
+  fontSize: '$0',
+  padding: '$2 $3',
+  cursor: 'default',
+  '&::before': {
+    boxSizing: 'border-box',
+  },
+  '&::after': {
+    boxSizing: 'border-box',
+  },
+
+  borderRadius: '$default',
+  backgroundColor: 'transparent',
+  border: 'none',
+  transition: 'background 0.5s',
+  '&:focus': {
+    background: '$grey4',
+    cursor: 'pointer',
+  },
+}
+
+const StyledContent = styled(Content, {
+  minWidth: 130,
+  backgroundColor: '$paper',
+  borderRadius: '$default',
+  padding: '$1',
+  boxShadow: '$2',
+})
+
+const StyledItemIndicator = styled(ItemIndicator, {
+  position: 'absolute',
+  left: 5,
+})
+
+const DEFAULT_TAG = 'div'
+
+/**
+ * StyledMenu base component
+ */
+const StyledMenu = styled(Root, {})
```

Comment:
Not required unless you are going to add some styles in there.

Reply:
Doesn't this add the stitches css prop to the component though? Someone might want to do e.g. `<Menu css={{marginTop: "$1"}}`

Reply:
Fair point - was thinking it would already be there but I suppose not as radix can be used with other styling solutions..